tests/integration/test_bbs_database.py

```diff
@@ -70,6 +70,13 @@ def test_bbs_database(tmp_path, setup_backend, jsons_path):
     parsed_files_dir.mkdir()
 
     all_input_paths = sorted(jsons_path.rglob("*.json"))
+
+    # 16e82ce0e0c8a1b36497afc0d4392b4fe21eb174.json and PMC7223769.xml.json are the
+    # same article. Currently, the population of the database is not incremental. So,
+    # the case where the same article is added two times is not handled yet. The patch
+    # below is to move forward until the incremental population is implemented.
+    all_input_paths = [x for x in all_input_paths if x.name != "PMC7223769.xml.json"]
+
```

Comment:
Interesting:) I guess there are multiple possible solutions too
* Your solution and potentially remove this `if` statement once we have a real `article_id` generating logic (currently it just uses the article title) and hope that after this the 2 articles are not going to be "identical" anymore
* untrack / remove one of these jsons from our repo
* inplace modify one of the jsons to have , after all this is just test data
* `pytest.raises` when we try to add an identical article the second time around

Anyway, I am totally fine with what you did here:)

Reply:
btw regarding your comment I would argue that we do have incremental population. However, we just happen to have two articles that are identical (=the same title under the current definition). I guess the fact we get an error when adding an already existing article is actually the desired behavior we agreed on. Or would you disagree?

Reply:
> hope that after this the 2 articles are not going to be "identical" anymore
This is the same article.

> after all this is just test data
I don't know why we have the same article in our test data. Maybe the others could tell us if there is a reason.

So, until then, I suggest to move on.

Reply:
> I would argue that we do have incremental population

The common understanding of "incremental population" implies that duplicates are handled (without crashing).

This is currently not the case as the code crashes (`IntegrityError` from MySQL).

Reply:
As discussed:
I have removed the term 'incremental' from the comment in 0e0a84e.

Reply:
Perfect, thank you very much!:)

Reply:
Regarding the definition of 'incremental population':
The idea is to handle duplicates at the Python code level. This means that `bbs_database add <database url> <directory path of parsed articles>` could run without stopping due to an exception (current situation).